android/app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,6 @@
 <resources>
     <string name="app_name">Go_social</string>
-    <string name="facebook_app_id">2349388348405699</string>
+    <string name="facebook_app_id">222891166977339</string>
+    <string name="fb_login_protocol_scheme">fb1234</string>
+    <string name="facebook_client_token">PLACE_YOUR_CLIENT_TOKEN_HERE</string>
```

Comment:
Added configuration in readme file

Reply:
Please review the **update reademe** pr [#169](https://github.com/scorelab/Go-social/pull/169), so that i can also update this comfig in new readme file.

Reply:
Please address the issues in #169 first.

Reply:
Sure, i will address this